packages/geo-diary-marketing/src/components/compatibility-slide.svelte

```diff
@@ -103,13 +109,13 @@
     font-size: 0.875rem;
   }
 
-  @media screen and (max-width: 1100px) {
+  @media screen and (max-width: 2880px) {
```

Comment:
variable?  non-response breakpoint?

Reply:
As discussed, you can't use css vars in media queries am afraid